File: CompSci 1/Unit 12, Selection/Compare/Compare.java
import java.util.Scanner;

public class Compare {
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter Number 1 --> ");
        int n1 = sc.nextInt();
        System.out.print("Enter Number 2 --> ");
        int n2 = sc.nextInt();

        if (n1 == n2) {
            System.out.println(n1 + " is equal to " + n2);
        }
        else if (n1 > n2) {
            System.out.println(n1 + " is greater than " + n2);
        }
        else if (n1 < n2) {
            System.out.println(n1 + " is less than " + n2);
        }
    }
}

File: CompSci 1/Unit 12, Selection/Compare/Compare.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    print("Enter Number 1 --> ")
    val n1 = sc.nextInt()
    print("Enter Number 2 --> ")
    val n2 = sc.nextInt()

    if (n1 == n2) {
        println(n1.toString() + " is equal to " + n2)
    } else if (n1 > n2) {
        println(n1.toString() + " is greater than " + n2)
    } else if (n1 < n2) {
        println(n1.toString() + " is less than " + n2)
    }
}
